### PR TITLE
Mission view

### PR DESCRIPTION
**Pull request by 96kevrud**
Tested and it works! (Henrik vi glömde att lägga till "enter","middle" eller "exit" förut haha

### CODE REVIEW

File: src/mdsd/controller/RobotController.java [hunk 1]
```diff
@@ -69,6 +90,7 @@ public synchronized void update(IRobot robot) {
         }
         if (mission.reachedGoal(robot)) {
             updateTravelMap(robot);
+            views.stream().forEach(v -> v.updateMission(robots.indexOf(robot), mission.getStringList()));
```
Comment (feffes) [inline src/mdsd/controller/RobotController.java:93]: should be run before updating the travel map?

**Review by feffes — CHANGES_REQUESTED** [review]
Merge conflicts :tada:

File: src/mdsd/controller/RobotController.java (hunk 1 above)
Comment (96kevrud) [reply src/mdsd/controller/RobotController.java:93]: it dont think it matters. updateTravelMap is logical and views.stream... is only for the view

Comment (96kevrud) [reply src/mdsd/controller/RobotController.java:93]: I have fixed all merge issues myself. I can merge and push

Comment (feffes) [reply src/mdsd/controller/RobotController.java:93]: go ahead

**Pull request merged**